kotlin/add-graphics-with-symbols/src/main/java/com/esri/arcgisruntime/sample/addgraphicswithsymbols/MainActivity.kt

```diff
@@ -0,0 +1,288 @@
+/*
+ * Copyright 2020 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgisruntime.sample.addgraphicswithsymbols
+
+import android.graphics.Color
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.esri.arcgisruntime.geometry.Point
+import com.esri.arcgisruntime.geometry.PointCollection
+import com.esri.arcgisruntime.geometry.Polygon
+import com.esri.arcgisruntime.geometry.Polyline
+import com.esri.arcgisruntime.geometry.SpatialReference
+import com.esri.arcgisruntime.mapping.ArcGISMap
+import com.esri.arcgisruntime.mapping.Basemap
+import com.esri.arcgisruntime.mapping.view.Graphic
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay
+import com.esri.arcgisruntime.symbology.SimpleFillSymbol
+import com.esri.arcgisruntime.symbology.SimpleLineSymbol
+import com.esri.arcgisruntime.symbology.SimpleMarkerSymbol
+import com.esri.arcgisruntime.symbology.TextSymbol
+import kotlinx.android.synthetic.main.activity_main.*
+
+
+class MainActivity : AppCompatActivity() {
+
+  private val wgs84 = SpatialReference.create(4236)
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_main)
+
+    mapView.apply {
+      // create a map with the BasemapType topographic and
+      // displayed it in this view
+      map = ArcGISMap(Basemap.Type.OCEANS, 56.075844, -2.681572, 11)
+    }
+
+    //create the graphics overlay
+    GraphicsOverlay().let {
+      //add the overlay to the map view
+      mapView.graphicsOverlays.add(it)
+      //add some buoy positions to the graphics overlay
+      addBuoyPoints(it)
+      //add boat trip polyline to graphics overlay
+      addBoatTrip(it)
+      //add nesting ground polygon to graphics overlay
+      addNestingGround(it)
+      //add text symbols and points to graphics overlay
+      addText(it)
+    }
+  }
+
+  /**
+   * Add Graphics for 4 Points to the map's graphics overlay.
+   *
+   * @param graphicOverlay the GraphicsOverlay which retains the Graphics
+   */
+  private fun addBuoyPoints(graphicOverlay: GraphicsOverlay) {
+    //define the buoy locations
+    val buoy1Loc = Point(-2.712642647560347, 56.062812566811544, wgs84)
+    val buoy2Loc = Point(-2.6908416959572303, 56.06444173689877, wgs84)
+    val buoy3Loc = Point(-2.6697273884990937, 56.064250073402874, wgs84)
```

Comment:
it looks to me like 2 y vals are (were) one decimal shorter than the other 2, and one x val is (was) also shorter than the rest of the x vals. Since the concern is how it lines up, I've simply padded with 0s to make things square, here and in those large methods down below.